src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -167,19 +167,27 @@ Do you want to start the scheduler service now? [Y/n]  y
 If a problem occurs, check output in /var/log/proactive/scheduler
 ....
 
+The installation script will create a symbolic link called `default` under the root path chosen for the installation that will point to the current version folder.
+
+Accordingly, the ProActive server installation can be accessed using `/opt/proactive/default` path.
+
 Once the server is up, ProActive Web Portals are available at `\http://localhost:8080/`. The user can access to any of the four portals using the default credentials: *admin/admin*
```

Comment:
Hi, during the installation process user is asked to set up admin password on Linux installation. 
Also note that before running install.sh there was a step 1 of the installation process to install dependences:
```
sudo apt update
sudo apt upgrade
sudo unattended-upgrade -d
wget https://raw.githubusercontent.com/ow2-proactive/utility-scripts/main/dependencies-scripts/install-dependencies-debian.sh
chmod +x install-dependencies-debian.sh
./install-dependencies-debian.sh
```
shouldn't this be also documented or interlinked in a case it is documented somewhere else?

Reply:
@ankicabarisic installing dependencies is specific to nebulous

And no, we are not documenting for now all installation questions and answers, maybe in the future.